tensorflow-sources.file

```diff
@@ -11,6 +11,9 @@ BuildRequires: bazel java-env git
 %if "%{?enable_tf_mkldnn:set}" != "set"
 %define  enable_tf_mkldnn 1
 %endif
+%if "%{?enable_tf_mkldnn_stack_check:set}" != "set"
```

Comment:
@gartung , in the `build_options.py` you have used `tf_mkldnn_stack_trace` and here you are checking for `enable_tf_mkldnn_stack_check` .. should it be `enable_tf_mkldnn_stack_trace` OR `tf_mkldnn_stack_check` in  `build_options.py` ?

Reply:
It is supposed to be `tf_mkldnn_stack_check`